src/gei/Commands/GenerateReclaimCSVCommand.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter.Commands
+{
+    public class GenerateReclaimCSVCommand : Command
```

Comment:
Based on C# 's naming guidelines CSV should camel cased --> GenerateReclaimCsvCommand`

Reply:
Done [896169](https://github.com/github/gh-gei/pull/365/commits/8961698de8f74966b45b04cbc31baee33cd9f756)